### PR TITLE
Simple examples of utilizing scss variables within styled components.…

### PR DESCRIPTION
**Pull request by devinleighsmith**
… Open to extension within a theme.

### CODE REVIEW

File: frontend/src/_variables.module.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import './colors.scss';
+@import './variables';
+
+:export {
```
Comment (Fosol) [inline frontend/src/_variables.module.scss:4]: Interesting that this syntax is importable within the js file.

**Review by Fosol — APPROVED** [review]

File: frontend/src/components/common/form/StepForm/SteppedForm.tsx [hunk 2]
```diff
@@ -66,7 +67,7 @@ const TabbedForm = styled(Form)`
     }
     border: 0;
     .btn:disabled {
-      background-color: #003366;
+      background-color: ${variables.primaryColor};
```
Comment (goemen) [inline frontend/src/components/common/form/StepForm/SteppedForm.tsx:70]: I like

**Review by goemen — APPROVED** [review]
Looks good to me

**Review by agalbrai — APPROVED** [review]
Super helpful thank you!

**Pull request merged**